hack/local-up-master/util.sh

```diff
@@ -70,7 +70,9 @@ kube::util::trap_add() {
 
 # Opposite of kube::util::ensure-temp-dir()
 kube::util::cleanup-temp-dir() {
-  rm -rf "${KUBE_TEMP}"
+  if [[ ! -z ${KUBE_TEMP-} ]]; then
+    rm -rf "${KUBE_TEMP}"
```

Comment:
Also this doesn't fail if there is no dir, so you don't technically need the check